index.js

```diff
@@ -83,7 +118,16 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  let movieID = null;
```

Comment:
No need to create this variable. We could simply return `null` after our `for` loop